docs/source/tools/traffic_vault_migrate.rst

```diff
@@ -0,0 +1,112 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _traffic_vault_migrate:
+
+#########################
+Traffic Vault Migrate
+#########################
+The ``traffic_vault_migrate`` tool - located at :file:`tools/traffic_vault_migrate/traffic_vault_migrate.go` in the `Apache Traffic Control repository <https://github.com/apache/trafficcontrol>`_ -
+is used to transfer TV keys between database servers. It interfaces directly with each backend so Traffic Ops/Vault being available is not a requirement.
+The tool assumes that the schema for each backend is already setup as according to the :ref:`admin setup <traffic_vault_admin>`.
+
+.. program:: traffic_vault_migrate
+
+Usage
+===========
+``traffic_vault_migrate [-cdhmr] [-f value] [-g value] [-o value] [-t value]``
+
+.. option:: -c, --compare
+
+		Compare 'to' and 'from' backend keys. Will fetch keys from the dbs of both 'to' and 'from', sorts them by cdn/ds/version and does a deep comparison.
+
+.. option:: -d, --dump
+
+		Write keys (from 'from' server) to disk in the folder 'dump' with the unix permissions 0640.
+
+		.. warning:: This can write potentially sensitive information to disk, use with care.
+
+.. option:: -f, --fromCfg=CFG
+
+		From server config file (default "riak.json")
+
+.. option:: -g, --toCfg=CFG
+
+		To server config file (default "pg.json")
+
+.. option:: -h, --help
+
+		Displays usage information
+
+.. option:: -o, --toType=TYPE
+
+		From server types (Riak|PG) (default "PG")
+
+.. option:: -m, --noConfirm
+
+		Do not require confirmation before inserting records
+
+.. option:: -r, --dry
+
+		Do not perform writes. Will do a basic output of the keys on the 'from' backend.
+
+.. option:: -t, --fromType=TYPE
```

Comment:
options that take option-arguments and have long and short forms should specify it for both forms e.g.
```rst
.. option:: -t TYPE, --from-type TYPE
```

The <kbd>=</kbd> can't be used with the short form, but using a space in the short form and an <kbd>=</kbd> in the long form doesn't hurt anything afaik.